applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -62,11 +62,17 @@ def model_part_reader(self, modelpart, nodeid=0, elemid=0, condid=0):
     def GetMainPath(self):
         return os.getcwd()
 
-    def __init__(self, model, parameters):
+    def __init__(self, model, DEM_parameters):
```

Comment:
why call it DEM_parameters?

Reply:
we are mixing different name variables to point the same thing. project_parameters, parameters and dem_parameters. Maybe we should consider some naming consistency.